src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerViewModel.cs

```diff
@@ -53,6 +53,12 @@ public PackageManagerViewModel(DynamoViewModel dynamoViewModel, PackageManagerSe
             PkgSearchVM.RegisterTransientHandlers();
 
             LocalPackages.CollectionChanged += LocalPackages_CollectionChanged;
+
+            // We are forced to make the udpate oureslves if the PreferencesViewModel has not been initialized yet
+            if (String.IsNullOrEmpty(PreferencesViewModel.SelectedPackagePathForInstall))
```

Comment:
If the PreferencesVM is not initialized, check the VM is not null before checking the SelectedPackagePathForInstall property.

Reply:
Also some typos in the comment.

Reply:
Addressed the comments, @reddyashish , it should be fine now!